crates/vm/levm/src/opcode_handlers/environment.rs

```diff
@@ -235,15 +233,21 @@ impl VM {
         }
 
         let mut data = vec![0u8; size];
-        for (i, byte) in current_call_frame
-            .bytecode
-            .iter()
-            .skip(code_offset)
-            .take(size)
-            .enumerate()
-        {
-            if let Some(data_byte) = data.get_mut(i) {
-                *data_byte = *byte;
+        if code_offset < current_call_frame.bytecode.len().into() {
+            let code_offset: usize = code_offset
+                .try_into()
+                .map_err(|_| VMError::VeryLargeNumber)?;
```

Comment:
Maybe this should be an internal error because it shouldn't ever fail. Right?

Reply:
Noted! Changed here: [374d904e9916f9b6559a6db3be259060893bc0f0](https://github.com/lambdaclass/ethrex/pull/1465/commits/374d904e9916f9b6559a6db3be259060893bc0f0)
Thanks Jere!